### ISSUE TITLE
Directly accessing a path does not work

### ISSUE THREAD

**Issue by shroukkhan** [UretzkyZvi/nextjs-typescript-tailwind-ionic-starter#3]
Trying to access a path directly ( for example : http://localhost:3000/tabs/feed ) return 404 error . Which also breaks the hot-reload. 

How to fix it?

**Comment by UretzkyZvi** [comment]
Hey @shroukkhan,

Thanks for bringing this to my attention! I've made a small update to help fix the problem. Now, when you try to go directly to any page under "/tabs", like "/tabs/feed", it should work just fine without showing a 404 error. This is because I've added a new rule that guides these specific requests back to the main page, and from there, the app knows where to take you.

The change I made 
 ```
/** @type {import('next').NextConfig} */
const nextConfig = {
  basePath: "",
  transpilePackages: ["@ionic/react", "@ionic/core", "ionicons"],
  output: "export",
  rewrites: () => {
    return [
      {
        source: "/tabs/:path*",
        destination: "/",
      },
    ];
  },
};

export default nextConfig;
 ```

Just a heads up - if you have other pages outside of "/tabs" that need the same fix, you'll need to add similar rules for those paths too.

Hope this helps and makes things smoother for you!
